roles/mirror_registry/vars/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Could you please delete this unused file

Reply:
Removed